doc/event-alarm-framework/event-alarm-framework.md

```diff
@@ -175,14 +167,11 @@ This modified file can then be uploaded to the device to /etc/evprofile/.
 Operator can select any of these custom event profiles to change default properties of events.
```

Comment:
Event profiling to change properties lie severity etc. would be performed by controllers as per their OAM policy.  Hence, can we remove the support to change the profile settings?